frontend/app/(dashboard)/layout.tsx

```
@@ -253,165 +209,73 @@ export default function DashboardLayout({ children }: { children: React.ReactNod
-);
-
-const NavLink = <T extends string>(props: LinkProps<T>) => {
-  const sidebar = useSidebar();
-  return <Link onClick={() => sidebar.setOpenMobile(false)} {...props} />;
-};
-
-const SupportUnreadCount = () => {
-  const { data } = useUnreadConversationsCount();
-  return data?.count && data.count > 0 ? <NavBadge count={data.count} /> : null;
-};
+};
```

ESLint

[error] Insert `⏎`

(prettier/prettier)